### PR TITLE
demonstrate verification key version

### PR DESCRIPTION
**Pull request by gurayAlsac**

### CODE REVIEW

File: internal/api/export/config.go [hunk 1]
```diff
@@ -29,12 +29,13 @@ var _ setup.DBConfigProvider = (*Config)(nil)
...
-	CreateTimeout time.Duration `envconfig:"CREATE_BATCHES_TIMEOUT" default:"5m"`
-	WorkerTimeout time.Duration `envconfig:"WORKER_TIMEOUT" default:"5m"`
-	MaxRecords    int           `envconfig:"EXPORT_FILE_MAX_RECORDS" default:"30000"`
-	Database      *database.Config
-	DefaultKeyID  string `envconfig:"EXPORT_FILE_DEFAULT_KEY_ID" default:"ExampleServer_k1"`
+	Port              string        `envconfig:"PORT" default:"8080"`
+	CreateTimeout     time.Duration `envconfig:"CREATE_BATCHES_TIMEOUT" default:"5m"`
+	WorkerTimeout     time.Duration `envconfig:"WORKER_TIMEOUT" default:"5m"`
+	MaxRecords        int           `envconfig:"EXPORT_FILE_MAX_RECORDS" default:"30000"`
+	Database          *database.Config
```
Comment (sethvargo) [inline internal/api/export/config.go:36]: Nit: let's move this to either the top or bottom of the struct while we're here.

**Comment by mikehelmick** [comment]
This is fine - but I'll undo this in #274 later

**Comment by mikehelmick** [comment]
/lgtm
/approve
/hold 

remove hold after addressing @sethvargo comment

**Pull request merged**